### DISCUSSION TITLE
Problem with re installed version

### DISCUSSION THREAD

**Discussion by Anel1210** [arkypita/LaserGRBL#1101] [Q&A]
Hi. My pc's hard drive crashed. Got it fixed and re-installed LaserGBRL. I do my designing and artwork with inkscape. I design my logos in such a way that I can break up the image into multiple images to do different types of engraving to get to the final product. Previously when I broke up the image into multiple parts, when I pulled it into LaserGBRL it kept the size ratio the same. Now after installing LaserGBRL again on the latest version available, it moves all the images to the left bottom corner and doesn't keep the aspect ratio as designed in my artwork. I don't know if the problem is with the values in the gbrl configuration codes or why it does it. Please could you assist as I have set the gbrl configuration values as previously. I can't remember which version was installed previously that's why I installed the latest version. I would appreciate your assistance. Regards Anel

**Comment by arkypita** [comment]
Many changes was made to SVG in v3.3.1 and between v3.1.4 / v3.1.5 to improve support for different SVG implementation.
What program do you use to generate SVG? Can you provide some file samples and required/vs obtained result?
